media-libs/sdl-pango/sdl-pango-0.1.2-r1.ebuild

```diff
@@ -28,6 +28,10 @@ src_unpack() {
 src_prepare() {
 	default
 	eapply -p0 "${DISTDIR}"/SDL_Pango-0.1.2-API-adds.patch
+
+	# Inspired by Fedora's SDL_Pango-0.1.2-fedora-c99.patch
+	sed -i -e '/SDL_Pango.h/i#define __FT2_BUILD_UNIX_H__' src/SDL_Pango.c || die
```

Comment:
Patch it instead? I don't really like sed for this because they're kind of smelly (they end up lurking forever even after they become obsolete, can do the wrong thing, etc). I know the risk here is low/non-existent but would appreciate it if you could.